scripts/apply_cal.py

```diff
@@ -17,6 +17,9 @@
 if args.vis_units is not None:
     kwargs['vis_units'] = args.vis_units
 
+if args.nbl_per_load == 0:
```

Comment:
Why is this necessary? When would this ever happen? Shouldn't this be an error?

Reply:
this is to allow for some flexibility in the makeflow bash interface so that we can use the same scripts for when we use partial i/o vs no partial i/o. 

0 is confusing though. I've changed this to "none".